code/src/layouts/header/components/header-toolbar/HeaderToolbar.tsx

```diff
@@ -81,18 +84,28 @@ const HeaderToolbar = () => {
           </AppBox>
         </AppBox>
         <AppBox className="header__toolbar-icons">
-          <AppIconButton>
-            <AppBadge
-              badgeContent={badgeContentTypography}
-              variant="dark"
-              size="small"
-            >
-              <ShoppingCartIcon
-                className="header__toolbar-cart-icon"
-                fontSize="large"
-              />
-            </AppBadge>
-          </AppIconButton>
+          <AppBox className="header__toolbar-action-icons">
+            <AppLink to={routes.orders.path}>
+              <AppIconButton>
+                <ListAltIcon
+                  className="header__toolbar-cart-icon"
+                  fontSize="large"
+                />
+              </AppIconButton>
+            </AppLink>
```

Comment:
Nesting `button` tag inside of `anchor` tag is a bad practice, it fails HTML validator test. I see that `AppIconButton` can be used as link itself by adding `to="your path"` and `component={AppLink}`.

Reply:
Yeah, you're right. Added forward ref to `AppLink` component. Now we can use it like `componen`t prop